src/axios.ts

```diff
@@ -86,52 +89,54 @@ export function multinetAxiosInstance(config: AxiosRequestConfig): MultinetAxios
     });
   };
 
-  Proto.searchUsers = function(query: string): AxiosPromise<UserSpec[]> {
-    return this.get('/user/search', {
+  Proto.searchUsers = function(username: string): AxiosPromise<UserSpec[]> {
+    return this.get('/users/search', {
       params: {
-        query,
+        username,
       },
     });
   };
 
-  Proto.tables = function(workspace: string, options: TablesOptionsSpec = {}): AxiosPromise<string[]> {
+  Proto.tables = function(workspace: string, options: TablesOptionsSpec = {}): AxiosPromise<Paginated<Table>> {
     return this.get(`workspaces/${workspace}/tables`, {
       params: options,
     });
   };
 
-  Proto.table = function(workspace: string, table: string, options: OffsetLimitSpec = {}): AxiosPromise<RowsSpec> {
-    return this.get(`workspaces/${workspace}/tables/${table}`, {
+  Proto.table = function(workspace: string, table: string, options: OffsetLimitSpec = {}): AxiosPromise<Paginated<TableRow>> {
+    return this.get(`workspaces/${workspace}/tables/${table}/rows`, {
       params: options,
     });
   };
 
-  Proto.graphs = function(workspace: string): AxiosPromise<string[]> {
-    return this.get(`workspaces/${workspace}/graphs`);
+  Proto.graphs = function(workspace: string): AxiosPromise<Paginated<Graph>> {
+    return this.get(`workspaces/${workspace}/networks`);
   };
 
   Proto.graph = function(workspace: string, graph: string): AxiosPromise<GraphSpec> {
-    return this.get(`workspaces/${workspace}/graphs/${graph}`);
+    return this.get(`workspaces/${workspace}/networks/${graph}`);
   };
 
-  Proto.nodes = function(workspace: string, graph: string, options: OffsetLimitSpec = {}): AxiosPromise<NodesSpec> {
-    return this.get(`workspaces/${workspace}/graphs/${graph}/nodes`, {
+  Proto.nodes = function(workspace: string, graph: string, options: OffsetLimitSpec = {}): AxiosPromise<Paginated<TableRow>> {
+    return this.get(`workspaces/${workspace}/networks/${graph}/nodes`, {
       params: options,
     });
   };
 
   Proto.attributes = function(workspace: string, graph: string, nodeId: string): AxiosPromise<{}> {
-    return this.get(`workspaces/${workspace}/graphs/${graph}/nodes/${nodeId}/attributes`);
+    return this.get(`workspaces/${workspace}/networks/${graph}/nodes/${nodeId}/attributes`);
   };
```

Comment:
This function doesn't exist in the Girder4 backend, so we should remove this.

Reply:
I left it because I wasn't sure if there were any plans to add it back later, but it has now been removed.

See 439d275